frontend/src/app/app.component.scss

```diff
@@ -1,83 +1,103 @@
+@import "src/utils/variables";
+
+$cards: now, delegate, own, trash;
+$sides: done, backlog;
+
+// Colors
+$button: #ecc31a;
+$cardBackground: rgba(66, 66, 66, 0.8);
+$sideBackground: rgba(255, 255, 255, 0.5);
+$cardText: rgb(255, 255, 255);
+$sideText: rgb(66, 66, 66);
+
 #appContainer {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-around;
-  background-color: #205159;
-  min-height: 100vh;
-  max-width: 100vw;
-  flex-flow: row wrap;
+  display: grid;
+  width: 100%;
+  height: 100%;
+  padding: $spacing;
+  box-sizing: border-box;
+  grid-template-columns: repeat(4, minmax(0, 1fr));
+  grid-template-rows: repeat(2, minmax(0, 1fr));
+  grid-template-areas:
+    "done now own backlog"
+    "done delegate trash backlog";
+  grid-gap: $spacing;
 
-  .flexCol {
-    flex-grow: 1;
-    padding: 20px;
-    width: 20vw;
+  @media (max-width: 800px) {
+    height: 200%;
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+    grid-template-rows: repeat(4, minmax(0, 1fr));
+    grid-template-areas:
+      "now own"
+      "delegate trash"
+      "backlog backlog"
+      "done done";
   }
-}
-
-.flexContainer{
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    overflow-x: hidden;
-}
 
-.titleText{
-    font-size: 20pt;
-    padding-top: 1rem;
-    padding-bottom: 1rem;
-}
+  @media (max-width: 800px) {
+    grid-template-columns: repeat(1, minmax(0, 1fr));
+    grid-template-rows: repeat(6, minmax(0, 1fr));
+    grid-template-areas:
+      "now"
+      "own"
+      "delegate"
+      "trash"
+      "backlog"
+      "done";
```

Comment:
Should we may not display this column on small windows / screen sizes? It is not a really helpful one.

Reply:
You mean the done "card"?

Reply:
Yes

Reply:
On mobile it is at the last card. Each card gets 1/3 of the Screenheight, so to see "trash" "backlog" and "done" you need to scroll down. I think its the best way, the important Cards are in the viewport for the Rest just scroll

Reply:
New Issue